.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Once this PR's updated from the latest master of this project, all `.idea/` based files in this PR should hopefully be ignored as they're mainly editor configurations.